src/session.ts

```diff
@@ -378,13 +383,24 @@ export class Session extends common.GrpcServiceObject {
     const reqOpts = {
       name: this.formattedName_,
     };
+
```

Comment:
This code looks repetitive, can't we add a function for this such as `header=get_header(this.parent.parent.parent as Spanner, this.resourceHeader_)`

Reply:
Done.